components/ResourceHub/company-resources.json

```
@@ -77,5 +77,18 @@
     ],
     "image": "https://i.ytimg.com/vi/h40Gy3fsRJg/hqdefault.jpg",
     "lastChecked": "2024-08-09"
+  },
+  {
+    "name": "Building with the Safe 4337 Module",
+    "url": "https://www.youtube.com/watch?v=VInxAVoYYkE",
+    "type": "Video",
+    "date": "2024-08-14",
+    "description": "The workshop focuses on the Safe 4337 Module and covers the process of building decentralized applications (dApps) using this module.",
+    "tags": [
+      "Safe Smart Account",
+      "4337"
+    ],
```

Adding "Safe{Core} SDK" as well:
```suggestion
      "4337",
      "Safe{Core} SDK"
    ],
```